README.md

```diff
@@ -1,19 +1,58 @@
 # mozilla-infobar
 
+This is the home of the Mozilla, Firefox update, and translation bars. These bars, in the first case, provides an easy way to let your users know that there is a newer version of Firefox available, and that they should upgrade.
+
+In the second instance, it enables you to notify your users when the current page they are viewing, is available in their preferred language.
+
+## Getting the InfoBar
+
+There are three ways to get the code.
+
+### Using Git
+
+First step is to clone the project.
+
+``` bash
+git clone git@github.com:mozilla/mozilla-infobar.git
+```
+
+Once the above operation has finished, you need to copy the following files to your project in the appropriate directories.
+
+``` bash
+css/infobar.css
+js/infobar.js
+```
+
+### Using NPM
+
+You can install it as a NPM module
+
+``` bash
+npm install mozilla-infobar
```

Comment:
Hmm yeah, but a CSS file doesn't need anything special to be able to require-able? The standard for JS published on npm seems to be CommonJS support for things that can run in the browser (unless I'm behind the times :smile:). If this kind of support isn't planned, would it be worth adding some extra info to the docs explaining how people should use the library when using npm specifically?